src/stomp_moveit_planning_context.cpp

```diff
@@ -105,7 +113,10 @@ bool StompPlanningContext::solve(planning_interface::MotionPlanResponse& res)
 
   // STOMP config, task, planner instance
   const auto group = getPlanningScene()->getRobotModel()->getJointModelGroup(getGroupName());
-  const auto config = getStompConfig(params_, group->getActiveJointModels().size() /* num_dimensions */);
+  auto config = getStompConfig(params_, group->getActiveJointModels().size() /* num_dimensions */);
+  robot_trajectory::RobotTrajectoryPtr input_trajectory;
+  if (extractSeedTrajectory(request_, input_trajectory) && !input_trajectory->empty())
```

Comment:
We don't have to implement this now, but it would be great if you could leave a note that failed seed trajectories should be handled in a special way, and not result in a fresh motion plan.